pandas/tests/series/test_operators.py

```diff
@@ -1300,25 +1290,52 @@ def test_datetime_series_with_timedelta(self):
         assert_series_equal(result, expected)
         assert_series_equal(result2, expected)
 
-    def test_datetime_series_with_DateOffset(self):
+    def test_dt64series_add_tick_DateOffset(self):
         # GH 4532
         # operate with pd.offsets
-        s = Series([Timestamp('20130101 9:01'), Timestamp('20130101 9:02')])
-
-        result = s + pd.offsets.Second(5)
-        result2 = pd.offsets.Second(5) + s
+        ser = Series([Timestamp('20130101 9:01'), Timestamp('20130101 9:02')])
         expected = Series([Timestamp('20130101 9:01:05'),
                            Timestamp('20130101 9:02:05')])
+
+        result = ser + pd.offsets.Second(5)
         assert_series_equal(result, expected)
+
+        result2 = pd.offsets.Second(5) + ser
         assert_series_equal(result2, expected)
 
-        result = s - pd.offsets.Second(5)
-        result2 = -pd.offsets.Second(5) + s
+    def test_dt64series_sub_tick_DateOffset(self):
+        # GH 4532
+        # operate with pd.offsets
+        ser = Series([Timestamp('20130101 9:01'), Timestamp('20130101 9:02')])
         expected = Series([Timestamp('20130101 9:00:55'),
                            Timestamp('20130101 9:01:55')])
+
+        result = ser - pd.offsets.Second(5)
         assert_series_equal(result, expected)
+
+        result2 = -pd.offsets.Second(5) + ser
         assert_series_equal(result2, expected)
 
+        with pytest.raises(TypeError):
+            pd.offsets.Second(5) - ser
+
+    def test_dt64series_with_DateOffset_smoke(self):
+        # GH 4532
+        # smoke tests for valid DateOffsets
+        ser = Series([Timestamp('20130101 9:01'), Timestamp('20130101 9:02')])
+
+        # valid DateOffsets
```

Comment:
parameterize (I think you already have these classes as a fixture in offsets somewhere)